contrib/sles11/alarm-notify-basic.bash3.sh

```diff
@@ -499,14 +493,14 @@ send_syslog() {
 
         message="${prefix} ${status}: ${chart} ${value_string}"
 
-        if [ ${host} ] ; then
+        if [ "${host}" ] ; then
             logger_options="${logger_options} -n ${host}"
-            if [ ${port} ] ; then
+            if [ "${port}" ] ; then
```

Comment:
As above for space preceding `;`

```suggestion
            if [ "${port}" ]; then
```